### DISCUSSION TITLE
Mention to care for implicit defaults when changing control plane parameters

### DISCUSSION THREAD

**Discussion by pat-s** [mysticaltech/terraform-hcloud-kube-hetzner#1435] [Ideas]
One can change control place parameters via `control_planes_custom_config`. However, it is not mentioned that implicit defaults are set (e.g. `kube-controller-manager-arg: flex-volume-plugin-dir=/var/lib/kubelet/volumeplugins`). 

If one now sets an argument in `control_planes_custom_config`, the control plane becomes unhealthy.

It would be great to mention in the example `kube.tf` that one should first look up the already defined parameters by checking `/etc/rancher/k3s/config.yaml` on a control plane node.

An alternative could be to return the active values in a terraform output.

**Comment by mysticaltech** [comment]
Implemented in PR: https://github.com/mysticaltech/terraform-hcloud-kube-hetzner/pull/2077

Closing as addressed for the v3 workstream.
